tox.ini

```diff
@@ -15,8 +15,10 @@ python =
 [testenv]
 deps = -rrequirements.txt
 commands =
-    pytest
-
+    pytest --override-ini junit_family=xunit1 --junit-xml=./results/UnitTest/junit.xml \
+          --cov ./sdv \
```

Comment:
Shall we exclude sdv/proto out of coverage report?

Reply:
hmm not sure, I mean technically we did not write the files and therefore don't need to test them, but on the other hand they are the base for everything...

Reply:
@mml5bg @BjoernAtBosch @doosuu  what do you think about the generated proto files?

Reply:
I am ok to have integration test to verify existence of generated files, probably version or some particular content of them, but do we need include line and branch coverage of generated files - sounds strange :)

Reply:
For me it also feels strange to test generated source code files. This should be covered by testing the generator - in our case the protoc (grpc), where I would expect it to be well-tested already.

Reply:
I agree that is strange to test generated source code files, but existence of them or particular version would be good and enough.

Reply:
excluded the proto-directory from the coverage calculation and report